test/lint_test.rb

```diff
@@ -0,0 +1,59 @@
+require "bundler/setup"
+require "safe_yaml"
+require "minitest/autorun"
+require "net/http"
+
+describe "lint test" do
+  BANNED_WORDS = Regexp.new('\b(' + [
+    "GitHub App",     # They're all GitHub Apps
+    "bot",            # TODO: link to docs about why bot is not preferred
+    "automatically"   # They're all automatic
+  ].join('|') + ')\b', Regexp::IGNORECASE | Regexp::MULTILINE)
+
+  Dir.glob("_apps/*.md").each do |path|
+    describe path do
+      # Load frontmatter
+      data = SafeYAML.load_file(path)
+      fields = SafeYAML.load_file("test/frontmatter.yml")
```

Comment:
Shouldn’t this be moved outside of the loop so i'd doesn’t have to get loaded separately for each bot?